test/runnable/issue16995.d

```diff
@@ -0,0 +1,28 @@
+// REQUIRED_ARGS: -unittest
+// COMPILE_SEPARATELY
+// EXTRA_SOURCES: imports/module_with_tests.d
+
+import imports.module_with_tests;
+import core.exception: AssertError;
+
+shared static this()
+{
+    import core.runtime: Runtime;
+    Runtime.moduleUnitTester = () => true;
```

Comment:
Returning `bool` is deprecated, return a number instead.
See also: https://github.com/dlang/druntime/pull/1685